apps/frontend/src/app/(app)/inbox/page.tsx

```diff
@@ -2,7 +2,7 @@ import { InboxPage } from "@/src/components/Inbox/InboxPage"
 
 const Inbox: React.FC = () => {
   return (
-    <section className="h-full overflow-y-auto bg-background ml-[260px] p-16 text-secondary-foreground">
+    <section className="h-full overflow-y-auto bg-background ml-[160px] p-16 text-secondary-foreground">
       <InboxPage />
```

Comment:
Just add a TODO here for the ml-[160px], can look weird on different screens. I'll fix the layout for all the places